testData/goto/ExternClass1.hx

```diff
@@ -1,4 +1,4 @@
-import com.bar.FooExtern;
+import com.bar.Foo;
```

Comment:
Because it's not valid haxe code. It is not compiled.

The one possible way to access FooExtern is to 
`import com.bar.Foo.FooExtern;` or `import com.bar.Foo;` - it's by haxe lang design